xcs-ui/xcs-ui-mainwikipreferences/pom.xml

```diff
@@ -94,6 +94,20 @@
               <xpath>/xwikidoc/object/property/version</xpath>
               <value>${xcs.footer.version}</value>
             </transformation>
+            <!-- Specify different rights for XWiki.XWikiGuest -->
+            <transformation>
+              <artifact>org.xwiki.enterprise:xwiki-enterprise-ui-admin-user</artifact>
+              <file>XWiki/XWikiPreferences.xml</file>
+              <xpath>xwikidoc/object[property/users[text()="XWiki.XWikiGuest"]]/property/levels</xpath>
+              <value>none</value>
```

Comment:
I think the good fix is to completely remove the object. It could be done with the maven-replace-plugin:
See: https://github.com/xwikisas/xcs/blob/master/xcs-web/pom.xml#L278-278